### PR TITLE
auto-expand 200 response data, make schema 'expand' arrow bigger/greener

### PR DESCRIPTION
**Pull request by timabrmsn**

### CODE REVIEW

**Review by adelegard — APPROVED** [review]

**Review by alanag13 — DISMISSED** [review]

File: source/layouts/api-docs-layout.erb [hunk 1]
```diff
@@ -37,6 +38,12 @@
                         fontFamily: "'proxima-nova', sans-serif",
                         fontWeight: "700"
```
Comment (unparalleled-js) [inline source/layouts/api-docs-layout.erb:39]: Can `fontWeight` be an int?

Comment (timabrmsn) [reply source/layouts/api-docs-layout.erb]: yep

**Comment by timabrmsn** [comment]
Is this repo configured differently than our others? I didn't think adding a change after approval cleared the approval state.

**Review by unparalleled-js — APPROVED** [review]

**Comment by unparalleled-js** [comment]
> Is this repo configured differently than our others? I didn't think adding a change after approval cleared the approval state.

It is set up that way, yes.  I don't mind that personally, sometimes it is useful and other times it is not.

**Pull request merged**